geopyspark/geotrellis/rdd.py

```diff
@@ -731,6 +732,13 @@ def pyramid(self, start_zoom, end_zoom, resample_method=NEARESTNEIGHBOR):
         if (num_cols & (num_cols - 1)) != 0 or (num_rows & (num_rows - 1)) != 0:
             raise ValueError("Tiles must have a col and row count that is a power of 2")
 
+        if start_zoom:
+            start_zoom = start_zoom
```

Comment:
There isn't a really compelling reason to do it that way, it's just one of two implementations of that check.